### DISCUSSION TITLE
False Alerts from Automations

### DISCUSSION THREAD

**Discussion by shawnhaywood** [ufodone/envisalink_new#121] [General]
Hello!  I installed the Envisalink Refactored HACS integration in order to address isuses with the default HA integration.  The install was simple and straightforward. Unfortunately, I'm getting a false autmation trigger from the following automation about once or twice a day.  Previously, I never experienced this problem.  Can anyone recommend a better automation to notify me when my alarm system triggers?  I use a envisalink v3 and Honeywell Vista 20p.  When the automation triggers, I recieve a log enry stating "Home became unavailable".  The alarm did not go off, but seems to trigger the automation becuase the panel is unavailable (maybe going offline for a moment).

'''
alias: Alarm System Lights ON when Triggered
description: Turns lights on when alarm triggered
trigger:
  - platform: state
    entity_id:
      - sensor.home_keypad
    attribute: alarm
condition: []
action:
  - type: turn_on
    device_id: 9eea17aa772d0ee7fd3e37afaeea639c
    entity_id: switch.kitchen_lights
    domain: switch
  - type: turn_on
    device_id: 963d50c380bef0118a02f3d4f599475a
    entity_id: switch.hall_light
    domain: switch
  - type: turn_on
    device_id: 6ba772bf03fdd0bfedf42bed646ec0c6
    entity_id: switch.den_light
    domain: switch
  - service: notify.pushbullet_notify
    metadata: {}
    data:
      message: Home Alarm System is Going Off
      title: Home Security Alarm
  - service: notify.shelly_pushbullet
    metadata: {}
    data:
      message: Home Alarm System is Going Off
      title: Home Security Alarm
mode: single
'''

**Comment by shawnhaywood** [comment]
As a quick follow-up.  I'm going to remove the "attribute" option and set "To" option to "triggered".  Maybe this will help.  Open to ideas.

**Comment by ufodone** [comment]
I use something like the following to explicitly look for the alarm moving to the "on" state:

```
trigger:
  - platform: state
    entity_id: sensor.home_alarm_keypad
    attribute: alarm
    from: "off"
    to: "on"
```

> When the automation triggers, I recieve a log enry stating "Home became unavailable". 

That message is possibly indicating that the integration lost connection to your EVL device.  Older firmware versions of the EVLs had problems like this so you might check that you're on the latest firmware.  This integration has shown to be a little more sensitive to this issue than the stock one because when it sends requests to the EVL, it expects to get a response.
